src/components/LayoutHelpers.tsx

```diff
@@ -15,7 +15,7 @@ const Home: React.FC = () => {
     return null
   }
 
-  usePageTitle(strings.PAGE_TITLE_HOME)
+  usePageTitle('Layout Helpers')
```

Comment:
Localisation?

Reply:
It's just a dev page -- probably won't be kept.

Reply:
yeah it's a dev page, so i thought to leave it out of localisations